### DISCUSSION TITLE
Unknown Runtime Error

### DISCUSSION THREAD

**Discussion by Archibald-Haddock-1941** [cpeditor/cpeditor#1046] [Q&A] [answered]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Whenever I try to run a nested vector it gets executed but shows a runtime error.

I used this (```vector<vector<int>> vec(10, v);```) in a program but 
``[16:42:42] [Runner[1]] [Execution has started]
[16:42:42] [Runner[1]] [Execution for test case #1 has finished with non-zero exitcode -1073741511 in 191ms]``
![image](https://user-images.githubusercontent.com/98262176/150677571-f1e8c1c0-d2ba-4fc9-80da-23fb41172cda.png)

this error was shown.
I am using winlibs gcc 11.2.0 and compilation command is default.

### Expected Behavior

It was expected to execute normally which is happening with other IDEs.

### Steps To Reproduce

as stated in current behavior

### CP Editor Version

6.10.1

### OS

Windows 10 64 bit

### Are you willing to contribute to this?

- [ ] I'm ready to fix this (and will open a PR soon)
- [ ] I'd like to have a try (with the help of the maintainers)
- [X] No, thanks

### Anything else?

please help me out, it is urgent and I really need it for my upcoming exam.

**Comment by Archibald-Haddock-1941** [comment]
@ouuan can you help ?

**Reply by ouuan** [reply]
This is not likely to be caused by CP Editor. Without further information, I've no idea what the cause might be.

**Comment by ouuan** [comment]
Have you encountered unexpected RE in other situations besides using nested vectors?

**Reply by Archibald-Haddock-1941** [reply]
nope

**Reply by coder3101** [reply]
Can you run the produced binary with same compiler and flags aa used by cpeditor?

**Reply by Archibald-Haddock-1941** [reply]
![image](https://user-images.githubusercontent.com/98262176/150682598-e07c682f-c408-495d-8a06-435d64328f4c.png)
no

**Reply by Archibald-Haddock-1941** [reply]
but the same code is working fine in code::blocks

**Reply by coder3101** [reply]
So it seems a problem with the compiled binary. Use g++ or a compiler used by codeblocks.

**Reply by Archibald-Haddock-1941** [reply]
it worked with the one of code::blocks.
Thank you.

**Comment by coder3101** [comment, marked as answer]
It seems issue with compiler installation, maybe two compilers in path or something related which causes a compiler to use headers of other compiler. 

> So it seems a problem with the compiled binary. Use g++ or a compiler used by codeblocks.

**Reply by Archibald-Haddock-1941** [reply]
no it was that
it was due to [this](https://codeforces.com/blog/entry/98961#comment-877216)
currently i am using cpp14
codeblocks compiler worked bcz it was cpp11
can you get me any solution for cpp 20
sorry to disturb again

**Reply by coder3101** [reply]
> no it was that
> 
> it was due to [this](https://codeforces.com/blog/entry/98961#comment-877216)
> 
> currently i am using cpp14
> 
> codeblocks compiler worked bcz it was cpp11
> 
> can you get me any solution for cpp 20
> 
-std=c++20 pass to compiler for using C++20 standards provided your compiler version supports it.
